core/auth.py

```
@@ -232,11 +236,25 @@ def _save_keys(self) -> None:
+        default_key = secrets.token_urlsafe(32)
+        key_obj = APIKey(
+            key=default_key,
+            name="default-dev-key",
+            created_at=datetime.now(),
+            permissions={"read", "write"},
+            metadata={"environment": "development"}
+        )
+        self.api_keys[default_key] = key_obj
+        self.key_hashes[hashlib.sha256(default_key.encode()).hexdigest()] = default_key
+        self._save_keys()
+        self.logger.info(f"Created default API key for development")
```

Ruff (0.14.10)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)